### PR TITLE
[WIP] Full support of ONNX Round

### PR DESCRIPTION
**Pull request by mbencer**
Temporary solution is processed in PR: https://github.com/openvinotoolkit/openvino/pull/1605

### CODE REVIEW

File: ngraph/test/onnx/onnx_import.in.cpp [hunk 1]
```diff
@@ -2262,23 +2262,23 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_round)
...
-                                2.3f,
-                                2.5f,
-                                2.7f,
-                                -1.1f,
-                                -1.5f,
-                                -1.9f,
-                                -2.2f,
-                                -2.5f,
-                                -2.8f});
+    test_case.add_input<float>(
```
Comment (ilyachur) [inline ngraph/test/onnx/onnx_import.in.cpp:2265]: Why did you change input and output values for this test?

Comment (mbencer) [reply ngraph/test/onnx/onnx_import.in.cpp:2265]: @ilyachur I split the test to two parts: `onnx_model_round` contains test data supported by limited version https://github.com/openvinotoolkit/openvino/pull/1605 (which we think is acceptable as temporary solution). Rest of test data was moved to `onnx_model_round_half_nearest_even` test which should PASS in this PR.

Comment (mbencer) [reply ngraph/test/onnx/onnx_import.in.cpp:2265]: Reported bug because of failing on IE: 35631

**Comment by mbencer** [comment]
Full support was Round was introduced in https://github.com/openvinotoolkit/openvino/pull/2730
